tests/Types.php

```diff
@@ -2,20 +2,37 @@
 
 declare(strict_types=1);
 
-use ReactParallel\Runtime\Runtime;
 use ReactParallel\EventLoop\EventLoopBridge;
+use ReactParallel\Runtime\Runtime;
 
 use function PHPStan\Testing\assertType;
 
-$pool = Runtime::create(new EventLoopBridge());
+$runtime = Runtime::create(new EventLoopBridge());
+
+assertType('null', $runtime->run(static function (): void {
```

Comment:
Exact same as:

* https://github.com/reactphp-parallel/infinite-pool/pull/60#discussion_r1910057904
* https://github.com/reactphp-parallel/contracts/pull/15#discussion_r1910063286
* https://github.com/reactphp-parallel/event-loop/pull/63#discussion_r1910060798

`void` is not a value, and cannot be asserted upon, while `Closure(): void` is, and can be asserted upon.

You cannot evaluate a `void` expression, then assert upon it: it has no value by definition.

You will need to change this assertion:

```diff
-assertType('null', $runtime->run(static function (): void {
+assertType('Closure(): void', fn() => $runtime->run(static function (): void {
```